apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/BindingTests.cs

```diff
@@ -94,6 +101,35 @@ public async Task BindNull(SpannerDbType parameterType)
             }
         }
 
+        // [START spanner_test_query_bind_numeric_null]
+        // [END spanner_test_query_bind_numeric_null]
+        // [START spanner_test_query_bind_numeric_array_null]
+        // [END spanner_test_query_bind_numeric_array_null]
+        [Theory]
+        [MemberData(nameof(BindNullNumericData))]
+        public async Task BindNullNumeric(SpannerDbType parameterType)
+        {
+            // The emulator doesn't yet support the NUMERIC type.
```

Comment:
(Not sure if the previous comment was actually registered)

Use Skip.If here?

Reply:
Good catch. Done.